### ISSUE TITLE
CustomInput is broken

### ISSUE THREAD

**Issue by jonjes** [bestguy/sveltestrap#178]
It seems to be that the "id" field is the issue.  Perhaps it's not being passed down correctly into the component?

Error message:
```
ReferenceError: id is not defined
    at Object.CustomInput_svelte_instance.$$self.$$.update (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:117439)
    at init (https://sveltestrap.js.org/vendors~main.eb58e42bc4ae23b9a5e3.bundle.js:2:41151)
    at new CustomInput_svelte_CustomInput (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:119368)
    at Array.create_default_slot_19 (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:861049)
    at create_slot (https://sveltestrap.js.org/vendors~main.eb58e42bc4ae23b9a5e3.bundle.js:2:15289)
    at Array.FormGroup_svelte_create_else_block (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:150408)
    at FormGroup_svelte_create_fragment (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:152999)
    at init (https://sveltestrap.js.org/vendors~main.eb58e42bc4ae23b9a5e3.bundle.js:2:41226)
    at new FormGroup_svelte_FormGroup (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:155633)
    at Array.Sample_svelte_create_default_slot (https://sveltestrap.js.org/main.eb58e42bc4ae23b9a5e3.bundle.js:1:883351)
```

You can also see the error message on the Sveltestrap documents page for CustomInput (https://sveltestrap.js.org/?path=/story/components--custominputs)

![image](https://user-images.githubusercontent.com/2365749/92287482-824dd000-eecf-11ea-9c05-a221761931bf.png)

**Comment by bestguy** [comment]
Ugg, sorry this has been whack-a-mole with simplifying id logic.   Should be corrected by #179 and will publish today

**Comment by bestguy** [comment]
Should be corrected in sveltestrap@3.7.7, thanks for the heads up

**Comment by jonjes** [comment]
Thanks so much for your help on this. Really nice library.
